theano/tensor/nnet/corr.py

```diff
@@ -131,20 +132,22 @@ def c_support_code_apply(self, node, nodename):
         if self.openmp:
             sub['omp_flags'] = '#pragma omp parallel for schedule(static)'
             sub['omp_max_threads'] = 'omp_get_max_threads()'
-            sub['omp_set_threads'] = 'omp_set_num_threads'
-            sub['omp_get_threads'] = 'omp_get_thread_num()'
+            sub['set_omp_threads'] = 'omp_set_num_threads'
+            sub['get_omp_threads'] = 'omp_get_thread_num()'
```

Comment:
The name change between the real omp fct name and the key make understanding the code a little bit harder. If you can reuse the omp name in the key, it would be good. But this is mostly a nitpik.